doc/api/dns.markdown

```diff
@@ -49,19 +46,20 @@ resolves the IP addresses which are returned.
       });
     });
 
-There are subtle consequences in choosing one or another, please consult the
-[Implementation considerations section][] for more information.
+There are subtle consequences in choosing one over the other another,
+please consult the [Implementation considerations section][] for more
+information.
 
 ## dns.getServers()
 
-Returns an array of IP addresses as strings that are currently being used for
-resolution
+Returns the array of string-encoded IP addresses that are currently being used
+for name resolution.
```

Comment:
YMMV and all that, but I'd go with something like this:

> Returns an array of IP addresses as strings that are being used for name resolution.

Or:

> Returns an array of strings. Each element in the array is an IP address that is being used for name resolution.

For me, the key changes are:

* `currently` is superfluous here because the verb is present tense, so let's take this opportunity to get rid of it.
* I agree with @silverwind that `string-encoded IP addresses` is fishy but I'm not sure `IP address strings` is better than the original `IP addresses as strings`. That last one seems like the idiomatic way to say what we're trying to say, at least to me.
* I also agree with @silverwind that `Returns an array` seems more natural than `Returns the array`.